pkgs/data/themes/sddm-sugar-dark/default.nix

```
@@ -0,0 +1,47 @@
+{ pkgs, lib, stdenvNoCC, themeConfig ? null }:
+
+let
+  name = "sugar-dark";
+  owner = "MarianArlt";
+
+  basePath = "$out/share/sddm/themes/${name}";
+  iniFormat = pkgs.formats.ini { };
+in
+
+stdenvNoCC.mkDerivation rec {
+  pname = "sddm-${name}";
+  version = "1.2";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "${owner}";
+    repo = "${pname}";
+    rev = "v${version}";
+    hash = "sha256-C3qB9hFUeuT5+Dos2zFj5SyQegnghpoFV9wHvE9VoD8=";
+  };
+
+  propagatedUserEnvPkgs = with pkgs.libsForQt5.qt5; [ qtgraphicaleffects ];
+
+  installPhase =
+    let
+      configFile = iniFormat.generate "" { General = themeConfig; };
+    in
+    ''
+      mkdir -p ${basePath}
+      cp -r $src/* ${basePath}
+    '' + lib.optionalString (themeConfig != null) ''
+      ln -sf ${configFile} ${basePath}/"theme.conf.user"
+    '';
+
+  meta = with lib; {
+    description = "The sugar-dark SDDM theme.";
+    longDescription = ''
+      Dark SDDM theme from the sugar family.
+      It is said to be so sweet, that it causes diabetes just from looking at it.
+    '';
+    homepage = "https://github.com/${owner}/${pname}";
+    license = licenses.gpl3;
+
```

```suggestion
    description = "Dark SDDM theme from the sugar family";
    homepage = "https://github.com/${owner}/${pname}";
    license = licenses.gpl3;
```
Please see the contributing guide for meta.description

---

Sorry I missed that. Applied your suggestion directly